### PR TITLE
Create Projects Area, Contact Form and Footer (Mobile First)

### PR DESCRIPTION
**Pull request by codedbystuart**
# Features added
* Projects List
* Contact Form
* Footer

## Files Changed
* index.html
* css/styles.css

### CODE REVIEW

**Review by awais-amjed — COMMENTED** [review]
Hi <STUDENT_MENTION>, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 

## Invalid Code Review Request

You have submitted a project, that has Linter issues 😃

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: css/styles.css [hunk 1]
```diff
@@ -12,12 +12,22 @@ a {
...
+}
+
 /* Header CSS */
 .header {
   padding: 12px 24px 12px 24px;
   background-color: #3c3a39;
 }
 
+.developer {
+  font-family: 'Roboto', sans-serif;
```
Comment (awais-amjed) [inline css/styles.css:28]: Kindly don't forget to import this font-family in order for it to work 😀

File: css/styles.css [hunk 2]
```diff
@@ -82,3 +92,251 @@ a {
...
+
+.projects-area {
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-rows: auto;
+}
+
+/* Project Card One */
+.project-card-one .proj-one-title {
+  font-family: 'Crete Round', serif;
```
Comment (awais-amjed) [inline css/styles.css:121]: Kindly don't forget to import this font-family in order for it to work 😀

File: css/styles.css [hunk 3]
```diff
@@ -82,3 +92,251 @@ a {
...
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  padding: 120px 24px 120px 24px;
+}
+
+.projects-title {
+  align-self: center;
+  padding-bottom: 24px;
+  font-family: 'Crete Round', serif;
```
Comment (awais-amjed) [inline css/styles.css:107]: Kindly don't forget to import this font-family in order for it to work 😀

File: css/styles.css [hunk 4]
```diff
@@ -82,3 +92,251 @@ a {
...
+/* Project Card Two */
+.project-card-two {
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url('/images/project-bg.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  padding: 24px 0 0 0;
+}
+
+.project-card-two .proj-two-title {
+  font-family: 'Crete Round', serif;
```
Comment (awais-amjed) [inline css/styles.css:166]: Kindly don't forget to import this font-family in order for it to work 😀

File: css/styles.css [hunk 5]
```diff
@@ -82,3 +92,251 @@ a {
...
+.aboutme {
+  padding: 0 24px 120px 24px;
+  background-color: #1c1a19;
+  background-image: url('/images/aboutme-bg.svg');
+  background-repeat: no-repeat;
+  background-position: 92% 28%;
+}
+
+.about-title {
+  font-family: 'Crete Round', serif;
```
Comment (awais-amjed) [inline css/styles.css:214]: Kindly don't forget to import this font-family in order for it to work 😀

File: index.html [hunk 6]
```diff
@@ -42,6 +39,145 @@ <h3 class="subtitle">I'am a Software Developer</h3>
...
+    <div class="language">
+      <h3 class="language-title"><i class="fa-solid fa-circle icon"></i> Skills</h3>
+      <ul class="languages">
+        <li class="lang">Codekit</li>
+        <li class="lang">Github</li>
+        <li class="lang">Codepen</li>
+        <li class="lang">Gitlab</li>
+        <li class="lang">Terminal</li>
+      </ul>
+    </div>
```
Comment (awais-amjed) [inline index.html:156]: Kindly use a `<ul>` tag with `<li>` tags inside instead of `<div>` tags for semantic seasons 😄

**Review by awais-amjed — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @codedbystuart , 

Good job so far! 👏👏 You have created an awesome design 🥳🧨🎈
There are some minor issues that you still need to work on to go to the next project but you are almost there! 🤗

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] Went overboard with the Requirements ✔️ 
- [x] Awesome Design ✔️
- [x] Good code organization ✔️ 

gif

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] I have noticed that you are trying to use Roboto font also but you forgot to import it. Kindly fix that too 😃
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (codedbystuart) [reply index.html]: @awais-amjed , do you mean use `<a>` tags inside the `<li>` tags?

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @codedbystuart  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (awais-amjed) [reply index.html]: @codedbystuart Not exactly. Even though you already got approved, those languages and skills and not supposed to be `<a>` tags. I am talking about this whole section being a list inside another list. Check the code below:

```
<ul>
  <li class="language">
    <h3 class="language-title"> <i class="fa-solid fa-diamond icon"></i> Language</h3>
    <ul class="languages">
      <li class="lang">Javascript</li>
      <li class="lang">Ruby</li>
      <li class="lang">Html</li>
      <li class="lang">Css</li>
    </ul>
  </li>
  <li class="language">
    <h3 class="language-title"> <i class="fa-solid fa-square icon"></i> Frameworks</h3>
    <ul class="languages">
      <li class="lang">Bootstrap</li>
      <li class="lang">Ruby On Rails</li>
      <li class="lang">RSPec</li>
      <li class="lang">CapyBara</li>
      <li class="lang">Selenium</li>
    </ul>
  </li>
  <li class="language">
    <h3 class="language-title"><i class="fa-solid fa-circle icon"></i> Skills</h3>
    <ul class="languages">
      <li class="lang">Codekit</li>
      <li class="lang">Github</li>
      <li class="lang">Codepen</li>
      <li class="lang">Gitlab</li>
      <li class="lang">Terminal</li>
    </ul>
  </li>
</ul>
```

Comment (awais-amjed) [reply index.html]: That is what I meant. Sorry for not explaining it correctly before 😃

**Pull request merged**